api_client/client.py

```diff
@@ -96,7 +96,7 @@ def _login(self, conn=None, headers=None):
         return {}
 
     def request(self, opt, content_type=DEFAULT_CONTENT_TYPE,
-                http_timeout=None, **message):
+                http_timeout=None, handle_errors=True, **message):
```

Comment:
it is better to name it with a more precious name e.g. unified_response, the flag is not only to control whether handle errors here

Reply:
raw_response is easier to understand. not sure what unified can mean as different devices have different response format.